src/shell/Aliases.ts

```diff
@@ -4,9 +4,12 @@ import * as _ from "lodash";
 export const aliasesFromConfig: Dictionary<string> = {};
 
 export async function loadAliasesFromConfig(): Promise<void> {
-    const lines = await executeCommandWithShellConfig("alias");
-
-    lines.map(parseAlias).forEach(parsed => aliasesFromConfig[parsed.name] = parsed.value);
+    try {
+        //const lines = await executeCommandWithShellConfig("alias");
+        //lines.map(parseAlias).forEach(parsed => aliasesFromConfig[parsed.name] = parsed.value);
+    } catch (error) {
+        console.error(error);
+    }
```

Comment:
I know about that, although I've decided to add only `cmd` support for now. I **love** PowerShell, but alternate shell support on Windows is another issue, and I'd like to limit scope of this issue to just `cmd.exe` and overall Windows support.